<ui version="4.0" >
 <class>ReceiptManage</class>
 <widget class="QWidget" name="ReceiptManage" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>813</width>
    <height>622</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Gesti√≥n de Recibos</string>
  </property>
  <property name="windowIcon" >
   <iconset resource="../resources/qzette.qrc" >
    <normaloff>:/app/images/app/receipt.png</normaloff>:/app/images/app/receipt.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3" >
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3" >
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2" >
       <item>
        <widget class="QFrame" name="frame" >
         <property name="frameShape" >
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow" >
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout" >
          <item>
           <widget class="QGroupBox" name="groupBoxDest" >
            <property name="title" >
             <string>Destino</string>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_2" >
             <item>
              <widget class="QRadioButton" name="buildingRadioButton" >
               <property name="text" >
                <string>Obra</string>
               </property>
               <property name="checked" >
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="extraRadioButton" >
               <property name="text" >
                <string>Extra</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_2" >
               <property name="orientation" >
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0" >
                <size>
                 <width>555</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </widget>
          </item>
          <item>
           <layout class="QGridLayout" name="gridLayout" >
            <item row="0" column="0" >
             <widget class="QPushButton" name="pb_buildingSearch" >
              <property name="text" >
               <string>Buscar &amp;Obra</string>
              </property>
              <property name="icon" >
               <iconset resource="../resources/qzette.qrc" >
                <normaloff>:/app/images/app/building.png</normaloff>:/app/images/app/building.png</iconset>
              </property>
             </widget>
            </item>
            <item row="0" column="1" >
             <widget class="QLabel" name="lb_building" >
              <property name="sizePolicy" >
               <sizepolicy vsizetype="Preferred" hsizetype="Expanding" >
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="frameShape" >
               <enum>QFrame::StyledPanel</enum>
              </property>
              <property name="text" >
               <string>Obra no seleccionada</string>
              </property>
              <property name="alignment" >
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item row="0" column="2" >
             <widget class="QComboBox" name="cb_building" >
              <property name="sizePolicy" >
               <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <item>
               <property name="text" >
                <string>-- Seleccione Obra --</string>
               </property>
              </item>
             </widget>
            </item>
            <item row="1" column="0" >
             <widget class="QPushButton" name="pb_employedSearch" >
              <property name="text" >
               <string>Buscar &amp;Empleado</string>
              </property>
              <property name="icon" >
               <iconset resource="../resources/qzette.qrc" >
                <normaloff>:/app/images/app/employed.png</normaloff>:/app/images/app/employed.png</iconset>
              </property>
             </widget>
            </item>
            <item row="1" column="1" >
             <widget class="QLabel" name="lb_employed" >
              <property name="sizePolicy" >
               <sizepolicy vsizetype="Preferred" hsizetype="Expanding" >
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="frameShape" >
               <enum>QFrame::StyledPanel</enum>
              </property>
              <property name="text" >
               <string>Empleado no seleccionado</string>
              </property>
              <property name="alignment" >
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item row="1" column="2" >
             <widget class="QComboBox" name="cb_employed" >
              <property name="sizePolicy" >
               <sizepolicy vsizetype="Fixed" hsizetype="Fixed" >
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <item>
               <property name="text" >
                <string>-- Seleccione Empleado --</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5" >
         <item>
          <widget class="QLabel" name="lb_search" >
           <property name="text" >
            <string>Buscar:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="le_filter" />
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer" >
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType" >
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0" >
        <size>
         <width>65</width>
         <height>19</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" >
     <item>
      <widget class="QTableView" name="tv_receipt" >
       <property name="editTriggers" >
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <property name="tabKeyNavigation" >
        <bool>false</bool>
       </property>
       <property name="alternatingRowColors" >
        <bool>true</bool>
       </property>
       <property name="selectionMode" >
        <enum>QAbstractItemView::SingleSelection</enum>
       </property>
       <property name="selectionBehavior" >
        <enum>QAbstractItemView::SelectRows</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line" >
       <property name="orientation" >
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="_2" >
       <item>
        <widget class="QToolButton" name="tb_add" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font" >
          <font>
           <pointsize>8</pointsize>
          </font>
         </property>
         <property name="text" >
          <string>Nuevo</string>
         </property>
         <property name="icon" >
          <iconset resource="../resources/qzette.qrc" >
           <normaloff>:/app/images/app/newreceipt.png</normaloff>:/app/images/app/newreceipt.png</iconset>
         </property>
         <property name="iconSize" >
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
         <property name="shortcut" >
          <string>F3</string>
         </property>
         <property name="toolButtonStyle" >
          <enum>Qt::ToolButtonTextUnderIcon</enum>
         </property>
         <property name="autoRaise" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="tb_view" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font" >
          <font>
           <pointsize>8</pointsize>
          </font>
         </property>
         <property name="text" >
          <string>Ver</string>
         </property>
         <property name="icon" >
          <iconset resource="../resources/qzette.qrc" >
           <normaloff>:/app/images/app/viewreceipt.png</normaloff>:/app/images/app/viewreceipt.png</iconset>
         </property>
         <property name="iconSize" >
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
         <property name="shortcut" >
          <string>F3</string>
         </property>
         <property name="toolButtonStyle" >
          <enum>Qt::ToolButtonTextUnderIcon</enum>
         </property>
         <property name="autoRaise" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="tb_remove" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font" >
          <font>
           <pointsize>8</pointsize>
          </font>
         </property>
         <property name="text" >
          <string>Eliminar</string>
         </property>
         <property name="icon" >
          <iconset resource="../resources/qzette.qrc" >
           <normaloff>:/app/images/app/removereceipt.png</normaloff>:/app/images/app/removereceipt.png</iconset>
         </property>
         <property name="iconSize" >
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
         <property name="shortcut" >
          <string>F5</string>
         </property>
         <property name="toolButtonStyle" >
          <enum>Qt::ToolButtonTextUnderIcon</enum>
         </property>
         <property name="autoRaise" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="Line" name="line_2" >
         <property name="orientation" >
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="tb_close" >
         <property name="sizePolicy" >
          <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font" >
          <font>
           <pointsize>8</pointsize>
          </font>
         </property>
         <property name="text" >
          <string>Ce&amp;rrar</string>
         </property>
         <property name="icon" >
          <iconset resource="../resources/qzette.qrc" >
           <normaloff>:/app/images/app/close.png</normaloff>:/app/images/app/close.png</iconset>
         </property>
         <property name="iconSize" >
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
         <property name="shortcut" >
          <string>F12</string>
         </property>
         <property name="toolButtonStyle" >
          <enum>Qt::ToolButtonTextUnderIcon</enum>
         </property>
         <property name="autoRaise" >
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer>
         <property name="orientation" >
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0" >
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>buildingRadioButton</tabstop>
  <tabstop>extraRadioButton</tabstop>
  <tabstop>pb_buildingSearch</tabstop>
  <tabstop>cb_building</tabstop>
  <tabstop>pb_employedSearch</tabstop>
  <tabstop>cb_employed</tabstop>
  <tabstop>le_filter</tabstop>
  <tabstop>tv_receipt</tabstop>
  <tabstop>tb_add</tabstop>
  <tabstop>tb_view</tabstop>
  <tabstop>tb_remove</tabstop>
  <tabstop>tb_close</tabstop>
 </tabstops>
 <resources>
  <include location="../resources/qzette.qrc" />
  <include location="../../QStationery/resources/qstationery.qrc" />
 </resources>
 <connections/>
</ui>
